sdk/documentintelligence/Azure.AI.DocumentIntelligence/tests/DocumentIntelligenceAdministrationClient/MiscellaneousOperationsLiveTests.cs

```diff
@@ -55,7 +55,11 @@ public async Task GetOperationWithDocumentModelBuild()
 
             OperationDetails operationDetails = await client.GetOperationAsync(disposableModel.Operation.Id);
 
-            var resourceLocation = $"{TestEnvironment.Endpoint}documentintelligence/documentModels/{disposableModel.ModelId}?api-version={ServiceVersionString}";
+            // The endpoint environment variable may or may not contain a trailing '/' character. Trim the string
+            // to ensure the behavior is consistent.
+
+            var trimmedEndpoint = TestEnvironment.Endpoint.Trim('/');
+            var resourceLocation = $"{trimmedEndpoint}/documentintelligence/documentModels/{disposableModel.ModelId}?api-version={ServiceVersionString}";
```

Comment:
Updating this in Document Intelligence as well since it's expected to fail once we enable live testing.

Reasoning: the endpoint I've been using locally ends with a `/` character, which does not seem to be the case for the one created in our live tests pipeline.
- Mine: https://<name>.cognitiveservices.azure.com/
- Pipeline: https://<name>.cognitiveservices.azure.com

I could just update my env var and remove the slash, but I believe this is likely to happen again if we leave it like this. Both formats are valid so it's fair to take both scenarios into account.